extensions/BMO/template/en/default/pages/user_activity.html.tmpl

```diff
@@ -104,6 +104,10 @@
   </p>
 [% END %]
 
+<p>
+  <a href="[% basepath FILTER none %]user_profile?user_id=[% user.id FILTER uri %]"> User Profile</a>
+</p>
```

Comment:
We need to deal with the possibility that multiple users were searched on instead of always coming from user_profile.html so we may need a loop here:

```
<p>
  User Profiles:
  [% FOREACH u = users %]
    <a href="[% basepath FILTER none %]user_profile?user_id=[% u.id FILTER uri %]">[% user.name FILTER html %]</a>[% ", " IF NOT loop.last() %]
  [% END %]
</p>